src/cloudWatchLogs/timeFilterSubmenu.ts

```diff
@@ -163,6 +188,27 @@ export class TimeFilterSubmenu extends Prompter<TimeFilterResponse> {
         return [new Date(parts[0]), new Date(parts[1])]
     }
 
+    /**
+     * Formats a given start and end time in milliseconds to
+     * a string in the format 'YYYY/MM/DD-YYYY/MM/DD'
```

Comment:
This "custom" case is the only datetime value that should be pre-filled. I don't think we should pre-fill the "relative" times because they are just a button-click.

Reply:
Sounds good, updated so it only does stuff with the ranges